git-cms-init

```diff
@@ -333,6 +333,10 @@ if [ ! -d "$CMSSW_BASE/src/.git" ]; then
   # update all branches and tags from the upstream repository
   git fetch official-cmssw --tags 2>&${verbose}
 
+  # create a new branch, pointing to the commit corresponding to $CMSSW_TAG, and switch to it
```

Comment:
NO, THIS IS NOT WHAT SHOULD BE DONE

Reply:
PLEASE DO NOT BREAK OTHER PEOPLE'S WORKFLOWS WITHOUT GOING THROUGH THE USUAL REVIEW PROCESS